plugins/out_stackdriver/stackdriver.h

```diff
@@ -61,15 +78,16 @@ struct flb_stackdriver {
     flb_sds_t token_uri;
     bool metadata_server_auth;
 
-    /* metadata server (GCP specific, WIP) */
-    flb_sds_t zone;
-    flb_sds_t instance_id;
-    flb_sds_t instance_name;
+    /* metadata server url */
+    flb_sds_t metadata_server;
 
-    /* other */
-    flb_sds_t resource;
     flb_sds_t severity_key;
 
+    /*
+     * A Stackdriver monitored resource and its labels
+     */
```

Comment:
For consistency, this should probably be a one-line comment.

Reply:
Done.